W-15428118-Accelerator for Service Cloud (#112)
mulesoft/docs-accelerators: service-cloud/1.0/modules/ROOT/pages/index.adoc

= MuleSoft Accelerator for Service Cloud
:service-cloud-version: {page-component-version}

> MuleSoft Accelerator for Service Cloud empowers service representatives with a holistic customer 360 view, facilitating personalized interactions, optimized service workflows, and heightened operational efficiency, all aimed at enhancing customer satisfaction.

Reviewer: ```suggestion
MuleSoft Accelerator for Service Cloud empowers service representatives with a holistic customer 360 view, facilitating personalized interactions, optimized service workflows, and heightened operational efficiency, to enhance customer satisfaction.
```

> == Use Cases

Reviewer: ```suggestion
== Use Case
```

> The following use cases are supported by the MuleSoft Accelerator for Service Cloud:

Reviewer: ```suggestion
The following use case is supported by the MuleSoft Accelerator for Service Cloud:
```

> * https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-service-cloud/minor/{service-cloud-version}/pages/Use%20case%201a%20-%20SAP%20to%20Salesforce%20Integration/[SAP to Salesforce Integration^] - Enable a single view of your customer by simplifying connectivity between SAP S/4 HANA and Salesforce.

Reviewer: ```suggestion
* https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-service-cloud/minor/{service-cloud-version}/pages/Use%20case%201a%20-%20SAP%20to%20Salesforce%20Integration/[SAP to Salesforce Integration]
+
Enable a single view of your customer by simplifying connectivity between SAP S/4HANA and Salesforce.
```
